cli/script/command-executor.ts

```diff
@@ -17,12 +17,15 @@ import * as Q from "q";
 import * as recursiveFs from "recursive-fs";
 var rimraf = require("rimraf");
 import * as semver from "semver";
+var simctl = require("simctl");
 import slash = require("slash");
 var Table = require("cli-table");
 import * as yazl from "yazl";
 var which = require("which");
 import wordwrap = require("wordwrap");
 
+var debugCommand = require("./commands/debug");
```

Comment:
put this in alphabetical order, right between `configFilePath` and `emailValidator`